### PR TITLE
Feature/verifying target object has all needed dependencies

### PR DESCRIPTION
**Pull request by vasilenkoigor**
I'm added case for verifying that target object has all needed dependencies. 
This is pull request for [4 issue](https://github.com/rambler-ios/RamblerTyphoonUtils/issues/4)

### CODE REVIEW

File: Code/AssemblyTesting/RamblerTyphoonAssemblyTests.m [hunk 1]
```diff
@@ -31,9 +31,16 @@ - (void)verifyTargetDependency:(id)targetObject
...
                        withClass:targetClass];
     
-    // Filtering the properties of the class
     NSMutableDictionary *allProperties = [[RamblerTyphoonAssemblyTestUtilities propertiesForHierarchyOfClass:targetClass] mutableCopy];
-    for (NSString *propertyName in [allProperties allKeys]) {
+    NSArray *propertyNames = [allProperties allKeys];
+    
+    // Verifying that target object has all needed dependencies
+    for (NSString *propertyName in dependencies) {
+        XCTAssertTrue([propertyNames containsObject:propertyName], @"У объекта %@ не было найдено свойство %@", targetObject, propertyName);
```
Comment (etolstoy) [inline Code/AssemblyTesting/RamblerTyphoonAssemblyTests.m]: We use English language in this project

**Comment by etolstoy** [comment]
@spbvasilenko thanks for your contribution! One small fix and we are done!

File: Code/AssemblyTesting/RamblerTyphoonAssemblyTests.m (hunk 1 above)
Comment (vasilenkoigor) [inline Code/AssemblyTesting/RamblerTyphoonAssemblyTests.m]: @igrekde Do you mean this comment? 
У объекта %@ не было найдено свойство %@

Comment (etolstoy) [inline Code/AssemblyTesting/RamblerTyphoonAssemblyTests.m]: Yes

Comment (vasilenkoigor) [inline Code/AssemblyTesting/RamblerTyphoonAssemblyTests.m]: Hmm...however, project has some Russian comments, like this: 

XCTAssertNotNil(dependencyObject, @"Свойство %@ объекта %@ не должно быть nil", propertyName, targetObject);

Comment (vasilenkoigor) [inline Code/AssemblyTesting/RamblerTyphoonAssemblyTests.m]: @igrekde ok, I'm fix it

**Comment by vasilenkoigor** [comment]
@igrekde In the last commit this is comment has been fixed

**Comment by etolstoy** [comment]
Great, today I'll release a new version!

**Comment by vasilenkoigor** [comment]
@igrekde Thank you!

**Pull request merged**